server/sonar-server/src/test/java/org/sonar/server/user/ws/CurrentActionTest.java

```diff
@@ -91,9 +92,9 @@ public void return_minimal_user_info() {
     CurrentWsResponse response = call();
 
     assertThat(response)
-      .extracting(CurrentWsResponse::getIsLoggedIn, CurrentWsResponse::getLogin, CurrentWsResponse::getName, CurrentWsResponse::getLocal,
+      .extracting(CurrentWsResponse::getIsLoggedIn, CurrentWsResponse::getLogin, CurrentWsResponse::getName, CurrentWsResponse::getAvatar, CurrentWsResponse::getLocal,
```

Comment:
Please do not check `CurrentWsResponse::getAvatar` but `CurrentWsResponse::hasAvatar` and expect the value to be `false`